services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] | None = None,
+        actors_ids: list[int] | None = None
+) -> list:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
Consider handling the `Movie.DoesNotExist` exception in `get_movie_by_id` to avoid potential errors when a movie with the given ID does not exist.